daemon/cmd/endpoint.go

```diff
@@ -315,6 +315,15 @@ func (d *Daemon) createEndpoint(ctx context.Context, owner regeneration.Owner, e
 		epTemplate.DatapathConfiguration.RequireRouting = &disabled
 	}
 
+	if option.Config.DisableSipVerification {
+		if epTemplate.DatapathConfiguration == nil {
```

Comment:
this condition might be checked twice in this function, perhaps consider moving it up?

Reply:
something like this?
```go
	if epTemplate.DatapathConfiguration == nil {
		epTemplate.DatapathConfiguration = &models.EndpointDatapathConfiguration{}
	}
	if option.Config.EnableEndpointRoutes {
		// Indicate to insert a per endpoint route instead of routing
		// via cilium_host interface
		epTemplate.DatapathConfiguration.InstallEndpointRoute = true

		// Since routing occurs via endpoint interface directly, BPF
		// program is needed on that device at egress as BPF program on
		// cilium_host interface is bypassed
		epTemplate.DatapathConfiguration.RequireEgressProg = true

		// Delegate routing to the Linux stack rather than tail-calling
		// between BPF programs.
		disabled := false
		epTemplate.DatapathConfiguration.RequireRouting = &disabled
	}

	if option.Config.DisableSipVerification {
		epTemplate.DatapathConfiguration.DisableSipVerification = true
	}
```
In the previous code the `EndpointDatapathConfiguration` was initialized **only** if `option.Config.EnableEndpointRoutes` is set to true, in the snippet above it will be initialized anyway. Is this ok?